File: python/FindUniueXOR.py
from enum import unique
import numbers


class Solution:
    def findUniue(nums):
        uni=0
        for i in nums:
            uni ^=i
        return uni

nums=[2,3,4,1,1,2,3,4,6]
x=Solution.findUniue(nums)
print(x)

File: typeScript/FindUniueXOR.ts
const findUnique=(nums:number[])=>{
    let unique=0;
    nums.forEach(element => {
        unique ^=element
    });
    return unique;
}

let nums=[1,3,4,4,3,1,2]
let x=findUnique(nums)
console.log(x);
